tests/robot-tests/tests/admin_and_public/bau/publish_data.robot

```diff
@@ -505,12 +505,17 @@ Approve release
     user approves original release for immediate publication
 
 Verify newly published release is on Find Statistics page
+    user waits for caches to expire
+
     user checks publication is on find statistics page    ${PUBLICATION_NAME}
 
 Navigate to published release page
     user clicks link    ${PUBLICATION_NAME}
     user waits until h1 is visible    ${PUBLICATION_NAME}    %{WAIT_MEDIUM}
 
+    user reloads the stale cached page
+    user waits until h1 is visible    ${PUBLICATION_NAME}    %{WAIT_MEDIUM}
```

Comment:
Doesn't seem like any of this is necessary as we're not doing an amendment, so the publication and release don't even exist beforehand?

Reply:
It's because the Publication slug URL is being hit rather than the direct Release one, hence we need to reload to get the latest Release's content.

Reply:
Not sure I understand. From what I can see in the test suite, neither the publication or release have been published, and we haven't previously visited the publication URL, so why would this even be cached?

Reply:
Hehe let me change it and give it a whirl!  I may have assumed too much here!

Reply:
By 'eck you were right!  Removed this line, ta